packages/flutter/lib/src/services/text_input.dart

```diff
@@ -931,6 +942,21 @@ class TextEditingValue {
     selection.hashCode,
     composing.hashCode,
   );
+
+  // Verify that the given range is within the text.
+  //
+  // This can't be perform during constructors, so perform this wherever else
+  // possible.
+  static bool _verifyRange(TextRange range, String text) {
```

Comment:
Nit: Is the bool useful here or should it just be void?

Reply:
Mostly for convenience. If we use `void` here we'll have to write
```dart
assert(() {
  _verifyRange(...);
  return true;
}());
```